spec/models/metric/ci_mixin/capture_spec.rb

```diff
@@ -36,6 +36,35 @@ def expected_stats_period_end
     parse_datetime('2013-08-28T12:41:40Z')
   end
 
+  context "#perf_capture_queue" do
+    it "splits up long perf_capture durations for old last_perf_capture_on" do
+      # there should be 10 days + a partial day, or 11 queue items
+      Timecop.freeze
```

Comment:
Yeah, I saw that and changed it in another update ... testing that out locally and I can update in a few minutes